documentation/book/getting-started.adoc

```diff
@@ -14,24 +14,9 @@ This guide expects that an {ProductPlatformName} cluster is available and the
 ifdef::Kubernetes[`kubectl` and]
 `oc` command-line tools are installed and configured to connect to the running cluster.
 
-.Supported Versions
-[cols="50%,50%",options="header"]
-|====
-|Product
-|Version
-
-ifdef::Kubernetes[]
-|{KubernetesLongName}
-|{KubernetesVersion}
-endif::Kubernetes[]
-
-|{OpenShiftLongName}
-|{OpenShiftVersion}
-
-|Apache Kafka
-|2.0.0
-
-|====
+ifdef::Downloading[]
```

Comment:
Should this be really `ifdef::Downloading[]`? I think that will render it only upstream. I think this should be `ifndef::Downloading[]` to show it only downstream, or?

Reply:
That's my understanding too. Those variables are confusingly named!

Reply:
So, this change should be **downstream** only.